search.go

```diff
@@ -0,0 +1,43 @@
+package filter
+
+import (
+	"gorm.io/gorm"
+)
+
+type Search struct {
+	Fields []string
+	Query  string
+}
+
+func (s *Search) Scopes(settings *Settings, modelIdentity *modelIdentity) func(*gorm.DB) *gorm.DB {
+	var fields []string
+
+	// Remove columns that not exist in the table
+	for _, field := range s.Fields {
+		_, ok := modelIdentity.Columns[field]
+		if ok {
+			fields = append(fields, field)
+		}
+	}
+
+	if len(fields) == 0 {
+		return nil
+	}
```

Comment:
We should leave this security because the fields defined by the developer in the parameters are just strings, the possibility of an error is quite large but adding an error message without blocking the execution of the request would be beneficial for the user but also for the developer who uses the project.
What do you think ?

Reply:
A warning would be nice indeed. Use `goyave.ErrLogger` to print something like "goyave.dev/filter warning: used non-existing column in search".

Reply:
Without this security, using a non-existing column will generate a database error so the developer will be aware of their mistake.
In a finished application, this kind of scenario should never happen so I think a crash is appropriate.